### PR TITLE
CLOUDP-284437: Add Spectral rule to ensure operationId uses camelcase

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-284437

### CODE REVIEW

File: tools/spectral/.spectral.yaml [hunk 1]
```diff
@@ -122,6 +122,15 @@ rules:
       functionOptions:
         match: "^[A-Za-z0-9-._~:/?#\\[\\]@!\\$&'()*+,;=]*$"
 
+  xgen-docs-operationId-camel-case:
+    description: operationId should be camelCased.
+    given: "$.paths[*]..operationId"
+    severity: error
+    then:
```
Comment (gssbzn) [inline tools/spectral/.spectral.yaml:129]: can you add a message like 
```suggestion
    message: "Operation ID should be camelCase: {{property}}."
    then:
```
so we can see what the current value is?

Comment (andreaangiolillo) [reply tools/spectral/.spectral.yaml:129]: good point! I will update it

Comment (andreaangiolillo) [reply tools/spectral/.spectral.yaml:130]: I updated to use `{{value}}` which gives the message "40221:26    error  xgen-docs-operationId-camel-case  Operation ID should be camelCase: grantMongoDBEmployeeAccess."

Using `{{property}}`` does not give a useful message  "40221:26    error  xgen-docs-operationId-camel-case  Operation ID should be camelCase: operationId."

Comment (gssbzn) [reply tools/spectral/.spectral.yaml:130]: perfect I wasn't sure what the right interpolation was, thanks for fixing it

**Review by lovisaberggren — APPROVED** [review]

**Review by blva — APPROVED** [review]

**Pull request merged**